src/main/java/uk/gov/digital/ho/hocs/audit/export/MalformedDateConverter.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.digital.ho.hocs.audit.export;
+
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.audit.export.adapter.DateAdapter;
+
+@Service
+public class MalformedDateConverter {
+
+    private DateAdapter dateAdapter = new DateAdapter();
+
+    public String[] correctDateFields(String[] auditData) {
```

Comment:
Medium: A null check is required for auditData and handled.

Reply:
done